pkg/rtspRecord/map_store.go

```diff
@@ -0,0 +1,63 @@
+package rtspRecord
+
+import (
+	"encoding/gob"
+	"github.com/edgenesis/shifu/pkg/logger"
+	"os"
+	"sync"
+)
+
+type PersistMap struct {
```

Comment:
Please add some comments on what does this structs do, and add some comments on the methods as well.